docs/nextflow_schema/nextflow_schema_examples.md

```diff
@@ -9,12 +9,12 @@ hide:
 You can see an example JSON Schema for a Nextflow pipeline `nextflow_schema.json` file below.
 
 This file was generated from the nf-core pipeline template, using `nf-core create`.
-It is used as a test fixture in the nf-validation package [here](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/nextflow_schema.json).
+It is used as a test fixture in the nf-schema package [here](https://github.com/nextflow-io/nf-schema/master/plugins/nf-schema/resources/nextflow_schema.json).
```

Comment:
sure about that url?

Reply:
Ah missed another one :grin: Thanks!

Reply:
Although no it's the correct one. (It works when I view it using `mkdocs serve`)

Reply:
All look ok to me for this PR, except this line, but if you say it's ok, it's a :heavy_check_mark: from me